ethcore/src/service.rs

```diff
@@ -31,6 +31,8 @@ pub enum SyncMessage {
 		good: Vec<H256>,
 		/// Hashes of blocks not imported to blockchain
 		bad: Vec<H256>,
+		/// Hashes of blocks that were removed from canonical chain
```

Comment:
in #645 `insert_block` returns retracted blocks and blocks that became canon. Will add more fields here when both are merged.